evm/cita-vm/Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM ewasm/bench-build-base:1
 
+LABEL maintainer="Ewasm Team"
+LABEL repo="https://github.com/ewasm/benchmarking"
+LABEL version="1"
+LABEL description="Ewasm benchmarking (cita-vm)"
```

Comment:
I know these files are not (yet) pushed to dockerhub, but I think it is fine to have these.